OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -53,6 +53,9 @@ public abstract class AttackBaseInfo : PausableConditionalTraitInfo
 		[VoiceReference]
 		public readonly string Voice = "Action";
 
+		[Desc("Facing to attack")]
+		public readonly WAngle AttackFacing = WAngle.Zero;
```

Comment:
Maybe this would be better named as something like `FiringOffset` or (better) `FiringAngle`? Since it is not really the facing the actor needs to have to attack but rather the facing/angle the weapon is at if I understand correctly.

Reply:
> I think ideally we wouldn't have that tolerance
Agree. 
And I just found `attack` in Attack(Activity) is AttackFrontal. That solve problem here.

> FiringOffset or (better) FiringAngle
good idea

Reply:
fixed